### DISCUSSION TITLE
Off week

### DISCUSSION THREAD

**Discussion by mihir-bombay-studio** [dear-digital/linter#89] [Announcements]
@anishdalvi @Mri1662 @kmalap05 @abhishekjani08 Due to some work I have to pause the internship next week until tuesday. We will pick up again exactly with the same points table from next week onwards. Thank you for your understanding.

**Comment by anishdalvi** [comment]
> @anishdalvi @Mri1662 @kmalap05 @abhishekjani08 Due to some work I have to pause the internship next week until tuesday. We will pick up again exactly with the same points table from next week onwards. Thank you for your understanding.

Roger that Sir🙌
